Automattic/wp-calypso
# Migrate media-library tests from enzyme to react testing library

#### Changes proposed in this Pull Request

This PR refactors the following test files to use `@testing-library/react` instead of `enzyme`.

- `client/my-sites/media-library/test/data-source.jsx`
- `client/my-sites/media-library/test/list-item-image.jsx`
- `client/my-sites/backup/backup-date-picker/test/index.jsx`

Part of #63409.

#### Testing instructions

Verify tests still pass: `yarn run test-client client/my-sites/media-library/ client/components/jetpack/daily-backup-status/`
